### ISSUE TITLE
Problem with renaming attachment

### ISSUE THREAD

**Issue by kafran** [jlegewie/zotfile#344]
If I select the check-box "Use Zotero to rename" the fields get unwritable. Also, on the tab "Advanced settings", "Automatically rename new attachments" = "**NEVER**" is not working, the attachments **always** gets renamed.

![image](https://user-images.githubusercontent.com/1889828/37430898-66d40f10-27b2-11e8-9955-0e79c1965714.png)

**Comment by duguyipiao** [comment]
The situation of mine is completely opposite. I also checked "Use Zotero to rename". But set "Automatically rename new attachments" = "always rename". The attachments NEVER get renamed.

Zotero 5.0.37, ZotFile, 5.0.7, MacOS 10.12.6

**Comment by e-clin** [comment]
Yeah, I'm fairly new to Zotero/Zotfile and agree that this set-up page is confusing - if I check "Use Zotero to Rename" then all settings below are greyed out and not modifiable. Zotfile is great, but I think the help documentation needs additional details - for example, to clarify whether checking this box means using Zotero instead of Zotfile for renaming.
